packages/devtools_app/lib/src/debugger/flutter/console.dart

```diff
@@ -52,20 +55,31 @@ class _DebuggerConsoleState extends State<DebuggerConsole> {
   Widget build(BuildContext context) {
     final numLines = _lines.length;
     final disabled = numLines == 0;
-    return Console(
-      lines: _lines,
-      controls: [
-        CopyToClipboardControl(
-          dataProvider: disabled ? null : () => _lines.join('\n'),
-          successMessage: 'Copied $numLines ${pluralize('line', numLines)}.',
-          buttonKey: DebuggerConsole.copyToClipboardButtonKey,
-        ),
-        DeleteControl(
-          onPressed: disabled ? null : widget.controller.clearStdio,
-          tooltip: 'Clear console output',
-          buttonKey: DebuggerConsole.clearStdioButtonKey,
-        ),
-      ],
+
+    return OutlineDecoration(
+      child: Column(
+        children: [
+          debuggerSectionTitle(Theme.of(context), text: 'Console'),
```

Comment:
Would it make sense that all Console widgets have a Title?

Reply:
I think it'd be legit to have title in all Consoles. Also, you could make the "debuggerSectionTitle" a "SectionTitle" so you can reuse on top of other widgets elsewhere, so the whole UI gains consistency.